src/packetizer/mod.rs

```diff
@@ -1,115 +1,108 @@
-use crate::error::Error;
-use crate::extension::abs_send_time_extension::*;
-use crate::header::*;
-use crate::packet::*;
-use crate::sequence::*;
+use crate::errors::RTPError;
+use crate::packet::Packet;
+use crate::sequence::Sequencer;
+use crate::{extension::abs_send_time_extension::*, header::Header};
 
-use std::io::{BufWriter, Read};
-use std::time::{Duration, SystemTime};
+use std::time::Duration;
 
-#[cfg(test)]
 mod packetizer_test;
 
 // Payloader payloads a byte array for use as rtp.Packet payloads
 pub trait Payloader {
-    fn payload<R: Read>(&self, mtu: isize, reader: &mut R) -> Result<Vec<Vec<u8>>, Error>;
+    fn payload(&self, mtu: u16, payload: &[u8]) -> Vec<Vec<u8>>;
 }
 
-// Packetizer packetizes a payload
-pub trait Packetizer {
-    fn packetize<R: Read, P: Payloader, S: Sequencer>(
-        &mut self,
-        reader: &mut R,
-        payloader: &mut P,
-        sequencer: &mut S,
-        samples: u32,
-    ) -> Result<Vec<Packet>, Error>;
+/// Packetizer packetizes a payload
+pub trait PacketizerInterface {
```

Comment:
still prefer Packetizer to PacketizerInterface

Reply:
Fixed